packages/atomic/cypress/plugins/index.js

```diff
@@ -15,7 +15,27 @@
 /**
  * @type {Cypress.PluginConfig}
  */
-module.exports = (_on, _config) => {
+module.exports = (on, _config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+
+  // auto open devtools in headed mode (does not appear in screenshots, recording)
```

Comment:
Personally I'd prefer no magic with the auto open dev tool (it's kind of annoying... ?) but if other prefer it then so be it !